frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -280,8 +278,8 @@ const Topology: React.FC<ComponentProps> = ({
       if (selectedIds.length && !visualization.getElementById(selectedIds[0])) {
         setSelectedIds([]);
       } else {
-        const selectId = getQueryArgument('selectId');
-        const selectTab = getQueryArgument('selectTab');
+        const selectId = queryParams.get('selectId');
+        const selectTab = queryParams.get('selectTab');
```

Comment:
`queryParams` should be a dependency.
We should remove the comment `// eslint-disable-next-line react-hooks/exhaustive-deps` because that is an old comment and the dependencies have changed since.

Reply:
✔️